### PR TITLE
Intly 9458

### PR DESCRIPTION
**Pull request by Rajagopalan-Ranganathan**
# Description
Had to create a new one because of permissions issue. Original PR: https://github.com/integr8ly/integreatly-operator/pull/1128
https://issues.redhat.com/browse/INTLY-9458

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test/common/alerts_exist.go [hunk 1]
```diff
@@ -396,6 +396,12 @@ var expectedRules = []alertsTestRule{
 			"RHMIInstallationControllerStoppedReconciling",
 		},
 	},
+	{
+		File: "redhat-rhmi-middleware-monitoring-operator-install-upgrade-alerts.yaml",
```
Comment (obrienrobert) [inline test/common/alerts_exist.go:400]: How about removing the hard coded Namespace prefix? The same way it's done for other alerts, e.g: `File: NamespacePrefix + "middleware-monitoring-operator-install-upgrade-alerts.yaml",`

Comment (Rajagopalan-Ranganathan) [reply test/common/alerts_exist.go:400]: Agreed. There is another constant, that suits the need here better: ```	NamespacePrefix                   = "redhat-rhmi-"
	RHMIOperatorNamespace             = NamespacePrefix + "operator"
	MonitoringOperatorNamespace       = NamespacePrefix + "middleware-monitoring-operator"``` . Changed it to MonitoringOperatorNamespace +

**Comment by jhellar** [comment]
Verified.
/lgtm

**Pull request merged**